stdlib/float.ml

```diff
@@ -292,6 +292,29 @@ module Array = struct
     in
     fill 0 l
 
+  (* duplicated from array.ml *)
+  let equal eq a b =
+  if length a <> length b then false else
+  try
+    for i = 0 to length a - 1 do
+      if eq (unsafe_get a i) (unsafe_get b i) then () else raise_notrace Exit
+    done;
+    true
+  with Exit -> false
+
+  (* duplicated from array.ml *)
+  let float_compare = compare
```

Comment:
I would keep the name `stdlib_compare` to make the "duplicated" comment exactly true :)

Reply:
But if someone changes `Float.compare` (see the source above) the variable name would become misleading.